modules/monitoring-setting-query-log-file-for-prometheus.adoc

```diff
@@ -0,0 +1,131 @@
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
++
+. Add `queryLogFile: <path>` for `prometheusK8s` under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      queryLogFile: <path> <1>
+----
+<1> The full path to the file in which queries will be logged.
++
+. Save the file to apply the changes.
++
+[WARNING]
+====
+When you save changes to a monitoring config map, pods and other resources in the related project might be redeployed. The running monitoring processes in that project might also be restarted.
+====
++
+. Check that the pods for the component are running. The following sample command lists the status of pods in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring get pods
+----
++
+. Read the query log:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring exec prometheus-k8s-0 -- cat <path>
+----
++
+[IMPORTANT]
+====
+Revert the setting in the config map after you have examined the logged query information.
+====
+
+** *To set the query log file for Prometheus in the `openshift-user-workload-monitoring` project*:
+. Edit the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
+----
++
+. Add `queryLogFile: <path>` for `prometheus` under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    prometheus:
+      queryLogFile: <path> <1>
+----
+<1> The full path to the file in which queries will be logged.
++
+. Save the file to apply the changes.
++
+[NOTE]
+====
+Configurations applied to the `user-workload-monitoring-config` `ConfigMap` object are not activated unless a cluster administrator has enabled monitoring for user-defined projects.
+====
++
+[WARNING]
+====
+When you save changes to a monitoring config map, pods and other resources in the related project might be redeployed. The running monitoring processes in that project might also be restarted.
+====
++
+. Check that the pods for the component are running. The following example command lists the status of pods in the `openshift-user-workload-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring get pods
+----
++
+. Read the query log:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring exec prometheus-k8s-0 -- cat <path>
+----
++
+[NOTE]
+====
+Revert the setting in the config map after you have examined the logged query information.====
```

Comment:
remove `====`

Reply:
this message is with `[NOTE]` icon, but the same message under below lines are with `[IMPORTANT]` icon
https://github.com/openshift/openshift-docs/pull/43449/files#diff-c934df74a3d8f522bb7fae910d0e0b4cb8f5a92facbe61d093a16297e8c9803dR73-R76

Reply:
Changed `[NOTE]` admonition to `[IMPORTANT]` for consistency